source/acquaman/IDEAS/IDEASXASScanConfiguration.h

```diff
@@ -31,7 +31,6 @@ class IDEASXASScanConfiguration : public AMStepScanConfiguration
 	Q_PROPERTY(QString edge READ edge WRITE setEdge)
 	Q_PROPERTY(double edgeEnergy READ energy WRITE setEnergy)
 	Q_PROPERTY(bool useFixedTime READ useFixedTime WRITE setUseFixedTime)
-	Q_PROPERTY(int numberOfScans READ numberOfScans WRITE setNumberOfScans)
 
```

Comment:
@dretrex I'm not sure about this changes. It should affect the database. But we really don't need it anymore.